doc/tutorials/enable_s5.rst

```diff
@@ -182,8 +183,8 @@ The procedure for enabling S5 is specific to the particular OS:
 #. If the Service VM is being shut down (transitioning to the S5 state), it can call
    ``acrnctl stop vm-name`` to shut down the User VMs.
 
-   .. note:: S5 state is not automatically triggered by a Service VM shutdown; this needs
-      to be run before powering off the Service VM.
+   .. note:: S5 state is not automatically triggered by a Service VM shutdown;
+      this command needs to be run before powering off the Service VM.
```

Comment:
"this command needs to be run before powering off the Service VM" is one of those subject-object mix-ups due to the passive voice. You can either be explicit about who or what powers off the VM, or you can go with the _total-passivity_ option ("this commend needs to be run before the Service VM is powered off").